### DISCUSSION TITLE
Why can't my computer connect to termux?

### DISCUSSION THREAD

**Discussion by MrAXUAN** [termux/termux-app#3543] [Q&A]
Device: OPPO A93s 5G
Termux version: 0.118.0

I installed OpenSSH on termux, but my computer doesn't connect

My phone with termux and computer are on the same wifi, and the computer cannot be connected through ssh, but I can connect again using juicessh on my phone with termux. What's going on?

**Comment by ChiefMikeK** [comment]
# :shipit:Also try asking: https://reddit.com/r/Termux
Have you tried going the other way ie. connect to computer from termux?  
Or setup a http server on your phone?

**Comment by jothi-prasath** [comment]
@MrAXUAN did you start the open ssh server, did set `passwd`
